### PR TITLE
Dev branch checking linters

### PR DESCRIPTION
**Pull request by lucaasdfg**
hey reviewer i hope you are having a good day, i just have one thing to tell you about the project, this image should be only one text but on the original page it looks like this https://archive.fo/UW4oR/7c09e7d0a510ee61a62275a115208926bbd7002a.png
so i tried to crop it but it didnt work so i leaved like that i hope you can help me out, good day :)

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work @cerfectus :clap: . However, some requirements needs to be implemented in order for us to approve your project. check below :arrow_down: 

#### Page structure

#### Navbar

* [x] It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section
- The buttons are non-interactive (not clickable)

image

- The input element is overflowing from the navbar. :pray: fix it

#### Bottom

* [x] It contains four elements which are links (to another articles or `#`)
> **You need to add a link to each div to make the div clickable**

#### Footer

* [x] It should be wrapped by `<footer>` tags

#### Implementation details

#### Background

- Use this [image](https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) for the main background image since you didn't find the original one :nerd_face: 

#### Navigation background

* [x] It uses CSS [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background
- **[REQUIRED]** You should set the background of your navigation bar to use gradient

#### General requirements 

* [ ] Add more info to you readme file (e.g, description of the project, technologies used, link to the assignment, link to you profiles, etc...).

- **[REQUIRED]** The linter is not properly configured it doesn't show any checks. Follow the instructions [here!](https://github.com/microverseinc/linters-config/tree/master/css).

### That's it :relaxed: . When finished, kindly resubmit another code review request :pray:

**Comment by lucaasdfg** [comment]
READ BEFORE PLEASEi already have the readme and solved the stickler file but i cant update it until i commit the merge and i know i need to wait for the code review before merging and this image is like this https://archive.fo/UW4oR/7c09e7d0a510ee61a62275a115208926bbd7002a.png so thats why it looks repeated in my code

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi @cerfectus! you have made progess :slightly_smiling_face: : 

But there points from my colleague,  you missed.

- [ ] Your README.md file is not detailed, make it more descriptive. Refer to this [link](https://github.com/cerfectus/Building-with-Backgrounds-and-Gradients/pull/1#pullrequestreview-292634668) Go to general requirements section on that file, to view is suggestions for a README.

**Also,**
- [ ]  The UI of your [clone](https://rawcdn.githack.com/cerfectus/Building-with-Backgrounds-and-Gradients/1f6d2adb16892455c28bdf4410b1acc67ebc19b9/index.html) doesn't look like what is obtained from [www.apple.com](https://web.archive.org/web/20140301004610/http://www.apple.com/) 

        -   Take your time and compare the two UI

#### Make the required changes, then submit another form for code review.

**Review by Forison — COMMENTED** [review]
# pull request under review by ADDO

File: index.html
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image ```<a>  </a>```

File: index.html
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image ```<a>  </a>```

File: index.html
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image ```<a>  </a>```

File: index.html
Comment (Forison) [inline index.html]: # use `````` tag for the image and wrap ```<a>``` around each image ```<a>  </a>```

**Review by Forison — CHANGES_REQUESTED** [review]
hello,👋
I have highlighted some issues that you have to fix.🙂 
* The four bottom images are links, thus image wrapped in an anchor tag.🙂 
* Take a second look at your background gradient implementation.🙂 
> implement the background-gradient on the ```nav``` ie the parent of the nav links.

####  **[optional]
check the UI of the original, and improve upon yours
## Kindly make those changes and submit for review again
Happy coding 🙂🙂🙂

**Review by stratospherique — APPROVED** [review]
### Great effort implementing the suggested fixes :clap: . Your project is approved :100: . You can merge to master :+1: :+1: 
###### Ps: resolve merging conflicts :mag:

**Pull request merged**
